### PR TITLE
Mission Computer and payload computer support

### PR DESCRIPTION
**Pull request by podhrmic**
Adding functionality for forwarding messages from the mission and payload computers to GCS and vice-versa. Related to https://github.com/paparazzi/pprzlink/pull/51

Also adds copilot module and adds minor changes to the way messages are parsed in the autopilot.

More details and the description of the system is here: https://wiki.paparazziuav.org/wiki/Mission_computer

### CODE REVIEW

File: sw/airborne/boards/chimera/chibios/v1.0/chimera.h [hunk 1]
```diff
@@ -142,7 +142,7 @@
 
 // AUX1
 #if USE_ADC_2
-#define AD1_2_CHANNEL ADC_CHANNEL_IN2
+#define AD1_2_CHANNEL ADC_CHANNEL_IN3
```
Comment (gautierhattenberger) [inline sw/airborne/boards/chimera/chibios/v1.0/chimera.h]: already fixed in master

File: sw/airborne/modules/mission/copilot.c [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+    /* Computes from (lat, long) in the referenced UTM zone */
+    struct LlaCoor_f lla;
+    lla.lat = RadOfDeg((float)(DL_MOVE_WP_lat(buf) / 1e7));
+    lla.lon = RadOfDeg((float)(DL_MOVE_WP_lon(buf) / 1e7));
+    lla.alt = ((float)(DL_MOVE_WP_alt(buf)))/1000.;
+    struct UtmCoor_f utm;
+    utm.zone = nav_utm_zone0;
+    utm_of_lla_f(&utm, &lla);
+    //nav_move_waypoint(wp_id, utm.east, utm.north, utm.alt);
```
Comment (gautierhattenberger) [inline sw/airborne/modules/mission/copilot.c]: please uncomment or remove

File: sw/airborne/modules/mission/copilot.c [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+bool send_cam_snapshot;
+bool send_cam_payload;
+bool send_copilot_status;
+
+struct CameraPayload cam_payload;
+struct CameraSnapshot cam_snapshot;
+struct CopilotStatus copilot_status;
+
+PPRZ_MUTEX(copilot_cam_snapshot_mtx);
```
Comment (gautierhattenberger) [inline sw/airborne/modules/mission/copilot.c]: Why are you using mutex ? There is no threads in this module.

File: sw/airborne/boards/chimera/chibios/v1.0/chimera.h (hunk 1 above)
Comment (podhrmic) [inline sw/airborne/boards/chimera/chibios/v1.0/chimera.h]: I think this came from `git merge master` - should I rebase or merge-rebase again?

File: sw/airborne/modules/mission/copilot.c (hunk 3 above)
Comment (podhrmic) [inline sw/airborne/modules/mission/copilot.c]: There might be threads in the future  - so just added protection and good practice. I would rather have an unused mutex than forget to add it later.

File: sw/airborne/modules/mission/copilot.c (hunk 2 above)
Comment (podhrmic) [inline sw/airborne/modules/mission/copilot.c]: Done.

**Comment by podhrmic** [comment]
Fixed - now there are no more conflicts

**Comment by podhrmic** [comment]
@gautierhattenberger are we good to merge this?

**Pull request merged**